tools/cockpit.spec

```diff
@@ -263,12 +263,17 @@ find %{buildroot}%{_datadir}/cockpit/selinux -type f >> selinux.list
 %endif
 
 %ifarch x86_64 %{arm} aarch64 ppc64le i686 s390x
+%if 0%{?fedora} || 0%{?rhel} < 8
 echo '%dir %{_datadir}/cockpit/docker' > docker.list
 find %{buildroot}%{_datadir}/cockpit/docker -type f >> docker.list
 %else
 rm -rf %{buildroot}/%{_datadir}/cockpit/docker
 touch docker.list
 %endif
+%else
+rm -rf %{buildroot}/%{_datadir}/cockpit/docker
+touch docker.list
+%endif
```

Comment:
This duplicates the `%else` branch, but as the `%if`s cannot be combined, I don't see a better way either.